test/testcafe/framework/page-objects/EnrollGoogleAuthenticatorPageObject.js

```diff
@@ -57,4 +57,8 @@ export default class EnrollGoogleAuthenticatorPageObject extends BasePageObject
   getCodeFieldError() {
     return this.form.getTextBoxErrorMessage(CODE_FIELD_NAME);
   }
+
+  getNextButtonClass() {
+    return this.form.getElement('.google-authenticator-next').getAttribute('class');
```

Comment:
if it is just to check the visibility, I think you can do something like
```
isNextButtonVisible() {
  return this.form.getElement('.google-authenticator-next').visible;
}
```
If we need to check the specific value for `display`, maybe we should specifically check for that 
```
getNextButton() {
 this.form.getElement('.google-authenticator-next');
}

// in the test
expect(getNextButton().getStyleProperty('display')).eql('block');
```